gdal_subsetter/transform.py

```diff
@@ -188,8 +182,9 @@ def process_item(self, item, source):
                     source, basename, input_filename, output_dir, layernames
                 )
             else:
-                logger.warn("No reconnized file format, not process")
-                return None
+                self.logger.warning('Will not process Unrecognised input file '
+                                    f'format, "{file_type}".')
+                raise UnknownFileFormatError(file_type)
```

Comment:
This is the second important change: previously an unrecognised file format was resulting in a successful request that had no output. This now will cause the request to fail, and state that it's because the service can't handle that type of input.